coalib/settings/ConfigurationGathering.py

```diff
@@ -22,24 +23,68 @@
                           '* add `-I` to suppress any use of config files\n')
 
 
-def aspectize_sections(sections):
+def aspectize_sections(sections, acquire_settings):
     """
     Search for aspects related setting in a section, initialize it, and then
     embed the aspects information as AspectList object into the section itself.
 
     :param sections:  List of section that potentially contain aspects setting.
     :return:          The new sections.
     """
-    for section_name, section in sections.items():
-        section.aspects = extract_aspects_from_section(section)
-        if section.aspects is not None and len(section.get('bears')):
-            logging.warning("'aspects' and 'bears' configuration is detected "
-                            "in section '{}'. Aspect-based configuration will "
-                            'takes priority and will overwrite any '
-                            'explicitly listed bears'.format(section_name))
+    for _, section in sections.items():
+        if verify_aspect_config(section, acquire_settings):
+            section.aspects = extract_aspects_from_section(section)
+        else:
+            section.aspects = None
     return sections
 
 
+def verify_aspect_config(section, acquire_settings):
```

Comment:
'verify_' is the same as 'check_' mentioned previously https://github.com/coala/coala/pull/4582#discussion_r135069786 
They both imply state is being checked, and no modifications are occurring.

Reply:
TBH I can't find any concise name/verb without it being too long (`verify_and_modify_aspect_config` doesnt seems good)...

Could you point a good name here?

Reply:
aspectize_section?